addons/phone_validation/models/res_partner.py

```diff
@@ -30,3 +36,15 @@ def _phone_format(self, number, country=None, company=None):
             force_format='INTERNATIONAL',
             raise_exception=False
         )
+
+    # FIXME DRY
+    def _phone_get_country_field(self):
+        if 'country_id' in self:
+            return 'country_id'
+        return False
+
+    # OVERRIDE
+    def _phone_get_number_fields(self):
```

Comment:
Is it used in any CRM flow without sms installed ?

Reply:
Our clients have call centers that would like to validate phone numbers, but do not text their customers.